.circleci/config.yml

```diff
@@ -758,7 +762,26 @@ workflows:
           compose_file: docker-compose-staging.yml
           stack_name: cl2-back-stg
           env_file: ".env-staging"
+
+  back-deploy-production:
+    when: << pipeline.parameters.back >>
+    jobs:
+      - back-build-docker-image:
```

Comment:
Maybe could just tag the master image (that is already built) with `production` instead of building a new image. This would make the production branch artificial, at least when it comes to the back-end, but would definitely be faster.

Reply:
I thought about that, but there is a risk of timing mismatch. The `master HEAD` could have changed from the moment this job runs, compared to when it was triggered (especially on failure and manual retry). Think it's best to keep the pipeline deterministic.
If we could, however, have the same SHA here than we had on master, we could just assume it already exists, but the merge commit adds another SHA to the head of `production` compared to master.